go.mod

```diff
@@ -8,14 +8,8 @@ require (
 	github.com/getlantern/hex v0.0.0-20190417191902-c6586a6fe0b7 // indirect
 	github.com/getlantern/hidden v0.0.0-20190325191715-f02dbb02be55 // indirect
 	github.com/getlantern/ops v0.0.0-20190325191751-d70cb0d6f85f // indirect
-	github.com/getlantern/ping v0.0.0-20191101185721-661eeb1317a8
 	github.com/go-stack/stack v1.8.0 // indirect
-	github.com/google/pprof v0.0.0-20191105193234-27840fff0d09 // indirect
-	github.com/ianlancetaylor/demangle v0.0.0-20181102032728-5e5cf60278f6 // indirect
 	github.com/sparrc/go-ping v0.0.0-20190613174326-4e5b6552494c
 	github.com/stretchr/testify v1.4.0
-	golang.org/x/arch v0.0.0-20191101135251-a0d8588395bd // indirect
-	golang.org/x/crypto v0.0.0-20191119213627-4f8c1d86b1ba // indirect
-	golang.org/x/sys v0.0.0-20191120155948-bd437916bb0e // indirect
-	golang.org/x/tools v0.0.0-20191121165004-82924fac8e73 // indirect
+	golang.org/x/net v0.0.0-20191209160850-c0dbc17a3553 // indirect
```

Comment:
I have no idea why this keeps getting in. Probably an implicit dependency of `errors` package?